src/patternfly/components/Table/table-scrollable.scss

```diff
@@ -7,6 +7,8 @@
   --pf-c-table__sticky-column--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-table__sticky-column--m-border-right--before--BorderRightWidth: var(--pf-global--BorderWidth--sm);
   --pf-c-table__sticky-column--m-border-right--before--BorderRightColor: var(--pf-global--BorderColor--100);
+  --pf-c-table__sticky-column--m-border-left--before--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-table__sticky-column--m-border-left--before--BorderLeftColor: var(--pf-global--BorderColor--100);
```

Comment:
Looks like the left modifier doesn't work because we have a typo in this var

https://github.com/patternfly/patternfly/blob/1e4fc83b2e8e9f8e86ad2c82684452c41a3bcba8/src/patternfly/components/Table/table.scss#L427

`--pf-c-table--cell--m-border-right--before--BorderLeftWidth` should be `--pf-c-table--cell--m-border-left--before--BorderLeftWidth`